docs/myst-overview.md

```diff
@@ -0,0 +1,136 @@
+# Overview about MyST - Markedly Structured Text
+
+## About
+
+> A Sphinx and Docutils extension to parse MyST, a rich and extensible flavour
+> of Markdown for authoring technical and scientific documentation.
+>
+> -- https://myst-parser.readthedocs.io/
+
+[MyST](https://myst-parser.readthedocs.io/) is a strict superset of the 
+[CommonMark syntax specification](https://spec.commonmark.org/).
+It adds features focussed on scientific and technical documentation authoring. {fas}`star2`
+
+
+### Details
+
+Roles and directives provide a way to extend the syntax of MyST in an unbound 
+manner, by interpreting a chunk of text as a specific type of markup, according
+to its name.
+Directives syntax is defined with triple-backticks and curly-brackets, effectively
+a Markdown code fence.
+
+-– [Roles and Directives](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html)
+
+**Example**
+
+> ![image|297x168](https://global.discourse-cdn.com/business7/uploads/crate/original/1X/d61bae27e7ce2ca04ab7ba78954153b885f2548e.png)
+
+
+## Overview
+
+### Inline quotes / citations
+
+> This text should be quoted.
```

Comment:
It looks like the markup instruction to quote/cite text does not render properly yet. /cc @msbt 

![image](https://github.com/crate/crate-docs-theme/assets/453543/2600ece5-199c-49e6-90c1-86b3465bc389)

-- https://crate-docs-theme--378.org.readthedocs.build/en/378/myst-overview.html#inline-quotes-citations

Reply:
Exactly. It is written like
```
> Franz jagt im komplett verwahrlosten Taxi quer durch Bayern.
```

and usually renders like
> Franz jagt im komplett verwahrlosten Taxi quer durch Bayern.

Reply:
I guess this is the issue:
- https://github.com/crate/crate-docs-theme/blob/main/src/crate/theme/rtd/crate/static/css/crateio-rtd.css#L27
- https://github.com/crate/crate-docs-theme/blob/main/src/crate/theme/rtd/crate/static/css/components.css#L187

Just a missing border, right?

Reply:
Looks good, I think you found the right spot, thanks. 🙇

Reply:
@matkuliak @amotl https://github.com/crate/crate-docs-theme/blob/main/src/crate/theme/rtd/crate/static/css/crateio-rtd.css#L27-L31 is indeed the main issue, if we remove those lines, the quote works. But if you have a paragraph wrapped in there, we might also need to override the `margin-bottom`, else it won't be centered vertically:

![image](https://github.com/crate/crate-docs-theme/assets/8767096/2d98ef1b-aa32-491d-aa73-3f00df26cba4)

Should I whip something up? And can you check if we're actually using the current `blockquote` somewhere in the docs @matkuliak?

Reply:
Thank you for submitting GH-380, aiming to fix this.

Reply:
I think this works well now, right?

- https://crate-docs-theme--378.org.readthedocs.build/en/378/rst/misc.html
- https://crate-docs-theme--378.org.readthedocs.build/en/378/myst/misc.html

Thank you again.